### PR TITLE
Hello_Microverse_project

### PR DESCRIPTION
**Pull request by omorich1**
In this branch we:

1. added a gitignore file
2. setup a lintel for HTML and CSS
3. added a readme file
4. added .html file
5. added .css file
6. add test.md file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
-*.agdai
-MAlonzo/**
+# test.md
```
Comment (carlos-ssh) [inline .gitignore]: - Add your `` node_modules `` to your ``.gitignore`` file.

File: ReadMe [hunk 2]
```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (carlos-ssh) [inline ReadMe]: Provide the right Name to your README file. it's missing style.

File: ReadMe [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### Prerequisites
+ Basic knowledge in HTML and CSS
+
+
+### Deployment
+
+Using an IDE to develop html and css code to modify it or run it directly with your browser 
+
+## Authors
```
Comment (carlos-ssh) [inline ReadMe]: - Provide information about how to dowload your project, install it locally and how to run your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @omorich1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,5 @@
+@@ -1,2 +1 @@
 *.agdai
 MAlonzo/**
+# test.md
```
Comment (SajjadAhmad14) [inline .gitignore]: Please uncomment `test,md`, and `node_modules` from `.gitignore`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+### Prerequisites
+ Basic knowledge in HTML and CSS
+
+### Setup
+Install a browser
+### Install
+
+### Usage
+
+### Run tests
```
Comment (SajjadAhmad14) [inline README.md]: Please write steps on how to install and use the project locally. Moreover, it seems that `Usage`, and `Run tests` are not used. Kindly remove unused sections from README as it does not look professional.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Richard Oveh, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .gitignore (hunk 1 above)
Comment (omorich1) [reply .gitignore]: uncommented node_modules and test.md

File: .gitignore (hunk 4 above)
Comment (omorich1) [reply .gitignore]: uncommented test.md and node_modules

File: README.md (hunk 5 above)
Comment (omorich1) [reply README.md]: included steps for installation and use of the project locally. I have also removed usage and run test that was not used please review

**Review by DanteAlonsoHT — APPROVED** [review]
# Status: Complete

Congratulations, I have bad internet right now, but I can see your project is complete, it's time to merge it!

**Review by DanteAlonsoHT — APPROVED** [review]
# Status: Complete

Congratulations, I have bad internet right now, but I can see your project is complete, it's time to merge it!

**Pull request merged**